src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup.Helpers/Conversions/JobConversions.cs

```diff
@@ -152,8 +160,41 @@ private static AzureRmRecoveryServicesAzureVmJobErrorInfo GetPSAzureVmErrorInfo(
             return psErrorInfo;
         }
 
+        private static string GetJobIdFromFullId(string fullId)
+        {
+            string[] splitArr = fullId.Split("/".ToCharArray());
+            return splitArr[splitArr.Length - 1];
+        }
+
+        #endregion
+
         #endregion
 
+        #region Enum translators
+
+        public static string GetJobTypeForService(BackupManagementType mgmtType)
+        {
+            switch (mgmtType)
+            {
+                case BackupManagementType.AzureVM:
+                    return "AzureIaasVM";
+                default:
+                    throw new Exception("Invalid BackupManagementType provided: " + mgmtType);
+            }
+        }
+
+        public static string GetPSBackupManagementType(string jobType)
+        {
+            if (jobType == "AzureIaasVM")
```

Comment:
I think there is one static class which has these values for jobType in hydra. pls try to use it or define constant